### ISSUE TITLE
Why did this project die?

### ISSUE THREAD

**Issue by IkariDevGIT** [JoshuaGrams/elm-story#1]
@JoshuaGrams why did this project die? i mean.. you had a userbase, you had a cool program. so.. why?

**Comment by JoshuaGrams** [comment]
Ah, so... this isn't my project. I wasn't even particularly following it. I don't have anything to do with the project other than that I happened to have a local copy of the repository so I put the code back up when the devs took it down.

They were pretty cryptic about it: I gather there was some sort of collaboration with another person (or company? maybe some sort of sponsor? or a dev partner? no idea) that went badly wrong.

But even before that... you say "you had a userbase" but they had been pretty clear (several times) that they did **NOT** have enough of a paying userbase that they could afford to continue putting significant amounts of time into it. And they had seemed pretty frustrated about that. So it had been kinda dead for a while anyway (maybe a year or more, IIRC).

**Comment by IkariDevGIT** [comment]
Sad to see this project dying, are we(the "community" behind it) allowed to continue with the source code without any licensing problems(or law problems smth like that)?

**Comment by JoshuaGrams** [comment]
Yes. It's GPL-v3 licensed. They can't take that away. That's the whole *point* of open-source and free-software licenses.
